src/platform/backends/lxd/lxd_virtual_machine.cpp

```diff
@@ -204,10 +204,6 @@ mp::LXDVirtualMachine::LXDVirtualMachine(const VirtualMachineDescription& desc,
 
 mp::LXDVirtualMachine::~LXDVirtualMachine()
 {
-    update_suspend_status = false;
-
-    if (current_state() == State::running)
-        suspend();
```

Comment:
(I know I answered a question about this differently before… sorry…)

Wouldn't shutting it down be more consistent? The only other platform that can't suspend (hyperkit) does that. I know, because we're the parent anyway - but leaving VMs running feels wrong.

Reply:
@Saviq,

Sure, I have no problem doing that.

Reply:
@Saviq,

Actually, having thought about it some more on a walk, I don't think stopping the instance on `multipassd` shutdown is the right thing to do.  If `snapd` decides to update Multipass (or any other reason the daemon may shutdown), I don't want whatever a user is doing in (an) instance(s) to be lost, so either we should leave it running or freeze it.

Reply:
IMO "`snapd` deciding to update" is much different than "any other reason"… The other reason for a clean shutdown would be `snap stop multipass`. We don't want VMs to stay on.

But for refresh, we can have a refresh hook that stores a file in `$SNAP_DATA` and we can NOOP on shutdown based on that. The launch script should then clear that file.

Reply:
Ok, I see what you mean.  I guess you're most concerned about leaving any `qemu` processes running when the intention is to shut down Multipass for some period of time, right?

It's difficult, from the daemon's perspective, to understand why it's being shut down.  The refresh stuff you suggest does help with that case.  However, I think that should be for a different PR and this PR will unilaterally shut down any running instances when the daemon exits.

Reply:
This is done.